src/components/button/Button.tsx

```diff
@@ -387,15 +408,16 @@ export function Button<T extends GroupTag = 'button'>({
   tag: _tag,
   themeOverride,
   type = 'button',
+  variant = 'standard',
   ...props
 }: ButtonProps<T>) {
   const tag = _tag || 'button';
   const renderContent = (_spinnerColor?: SpinnerProps['color']) => (
     <>
-      {!iconRight && icon}
-      {children}
-      {iconRight && icon}
-      {intent !== 'plain' && loading && (
+      <span className="content">{!iconRight && icon}</span>
+      <span className="content">{children}</span>
+      <span className="content">{iconRight && icon}</span>
```

Comment:
Could you try to change to another tag for these? `span` is an inline element, and `children` is possibly a `block` element. And having a block inside a inline is an invalid html standard 🤔 
```suggestion
      <span className="content">{!iconRight && icon}</span>
      <div className="content">{children}</div>
      <span className="content">{iconRight && icon}</span>
```